README.md

```diff
@@ -1,20 +1,26 @@
 # syscall_intercept
 
+
 [//]: # ([![Build Status]&#40;https://travis-ci.org/pmem/syscall_intercept.svg&#41;]&#40;https://travis-ci.org/pmem/syscall_intercept&#41;)
 
 [//]: # ([![Coverage Status]&#40;https://codecov.io/github/pmem/syscall_intercept/coverage.svg&#41;]&#40;https://codecov.io/gh/pmem/syscall_intercept&#41;)
 
 [//]: # ([![Coverity Scan Build Status]&#40;https://scan.coverity.com/projects/12890/badge.svg&#41;]&#40;https://scan.coverity.com/projects/syscall_intercept&#41;)
 
-This repository contains a multi-architecture porting of [syscall_intercept](https://github.com/pmem/syscall_intercept) working on both x86_64 and RISC-V
+This repository contains a multi-architecture porting of [syscall_intercept](https://github.com/pmem/syscall_intercept) working on both **x86_64** and **RISC-V**.
+Full-featured support for **aarch64** (ARM64) is **work in progress**.
 
+[![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
 # Dependencies #
 
+
 ## Runtime dependencies ##
 
  * libcapstone -- the disassembly engine used under the hood
    (RISC-V support requires version **5.0** or higher, while apt installs version 4.0.2 by default, as february 2025).
-   Compiling capstone by hand will require setting [CMakeLists.txt](CMakeLists.txt#L109) accordingly 
+   Since a manual installation of capstone could not be automatically detected while compiling syscall_intercept, we made
+   sure that libcapstone 5.0.6 is **automatically compiled** from source during the syscall_intercept cmake execution.
+   In other words, you don't need to care about having and installing capstone at all on your system.
```

Comment:
```suggestion
   Since a manual installation of Capstone could not be automatically detected while compiling syscall_intercept, we ensured that libcapstone v5.0.6 is **automatically compiled** from source during the syscall_intercept cmake execution.
```